modules/cnf-du-updating-performance-addon-operator.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-performance-addon-operator.adoc_{context}"]
+
+= Updating Performance Addon Operator
```

Comment:
After reading the title, expected to read about updating the Operator to the latest release.  However, that is already covered in [Updating Performance Addon Operator](https://docs.openshift.com/container-platform/4.7/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#upgrading-performance-addon-operator_cnf-master).  (Aside: my kingdom for a "the".)

Obligatory sugg: "Configuring a performance profile to support workload partitioning"--that's ugly.  Please revise the title somehow.